packages/design-system/src/components/scroll-area.tsx

```diff
@@ -32,15 +50,31 @@ const ScrollAreaScrollbar = styled(Scrollbar, {
   // ensures no selection
   userSelect: "none",
   // disable browser handling of all panning and scaleUp gestures on touch devices
-  touchAction: "none",
   padding: 2,
-  paddingRight: 3,
+  touchAction: "none",
   '&[data-orientation="vertical"]': {
     width: theme.spacing[6],
+    paddingRight: 3,
   },
   '&[data-orientation="horizontal"]': {
     flexDirection: "column",
     height: theme.spacing[6],
+    paddingBottom: 3,
+  },
+
+  variants: {
+    direction: {
+      both: {
+        "&[data-orientation=vertical]": {
+          marginBottom: theme.spacing[4],
+        },
+        '&[data-orientation="horizontal"]': {
+          marginRight: theme.spacing[4],
+        },
+      },
+      horizontal: {},
+      vertical: {},
```

Comment:
souldn't you have the same thing as in both but for horizontal and vertical?

Reply:
ah no, all good, this is space for when both are visible

Reply:
<img width="37" alt="image" src="https://github.com/webstudio-is/webstudio-builder/assets/5077042/ba502482-b7da-4f69-bc24-14c13359dc91">
Only this case need to be solved